src/api/kube.ts

```diff
@@ -189,7 +195,8 @@ export class KubeHelper {
       const options = new V1DeleteOptions()
       await k8sCoreApi.deleteNamespacedRole(name, namespace, options)
     } catch (e) {
-      throw new Error(e.body.message)
+      if (e.body && e.body.message) throw new Error(e.body.message)
```

Comment:
should we use helper method ? Instead of duplicating the same lines ?

Reply:
:+1: